Dockerfile

```diff
@@ -17,6 +17,7 @@
 # Build the manager binary
 # Run this with docker build --build-arg builder_image=<golang:x.y.z>
 ARG builder_image
+# hadolint ignore=DL3006
```

Comment:
Let's add a comment everywhere we've got an `ignore` explaining what we're ignoring and why.

Reply:
You're totally right about that. Added.